### PR TITLE
Added automatic version number from tag on README

### PR DESCRIPTION
**Pull request by og113**
Tiny change, following a similar one on WallGoMatrix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ WallGo is open-source scientific software for computing the bubble wall speed fo
 
 ## Status
 
-**Version** v0.1.0 (alpha)
+[![Version](https://img.shields.io/github/v/tag/Wall-Go/WallGo?label=Version)](https://github.com/Wall-Go/WallGo/tags/)
```
Comment (pschicho) [inline README.md:11]: Shouldn't this go to the `latest` release?

Comment (og113) [reply README.md:11]: I don't think we have any "releases" yet for this repo.

Comment (og113) [reply README.md:11]: If you can find a better url to point to, please suggest it.

Comment (pschicho) [reply README.md:11]: Right, let's update this when we have a release.

**Review by pschicho — APPROVED** [review]

**Pull request merged**